tools/meson.build

```diff
@@ -0,0 +1,37 @@
+libcomposefs_dep = declare_dependency(link_with : libcomposefs, include_directories : config_inc)
+
+thread_dep = dependency('threads')
+
+executable('mkcomposefs',
+    'mkcomposefs.c',
+    dependencies : [libcomposefs_dep, thread_dep],
+    install : true,
+)
+
+executable('mount.composefs',
+    'mountcomposefs.c',
+    dependencies : [libcomposefs_dep],
+    install : true,
+    install_dir : get_option('sbindir'),
+)
+
+executable('composefs-info',
+    ['composefs-info.c', '../libcomposefs/hash.c'],
+    c_args : ['-DUSE_OBSTACK=0', '-DTESTING=0', '-DUSE_DIFF_HASH=0'],
```

Comment:
We should be able to reference a variable for this to dedup with the top of https://github.com/containers/composefs/pull/300/files#diff-4f12b63e85afe8c8819073d64da906f24d8fe7ad05e0cc1db6fab193413e4410 right?